tests/automation/password.spec.ts

```diff
@@ -1,111 +1,174 @@
+import { test_Alice_1W_no_network } from './setup/sessionTest';
 import {
   clickOnMatchingText,
   clickOnTestIdWithText,
+  hasElementPoppedUpThatShouldnt,
   typeIntoInput,
   waitForTestIdWithText,
 } from './utilities/utils';
 
 const testPassword = '123456';
 const newTestPassword = '789101112';
 
-sessionTestOneWindow('Set Password', async ([window]) => {
-  // Create user
-  await newUser(window, 'Alice');
+async function expectRecoveryPhraseToBeVisible(
+  window: Page,
+  recoveryPhrase: string,
+) {
+  await waitForTestIdWithText(
+    window,
+    'recovery-phrase-seed-modal',
+    recoveryPhrase,
+    1000,
+  );
+}
+
+test_Alice_1W_no_network('Set Password', async ({ alice, aliceWindow1 }) => {
   // Click on settings tab
-  await clickOnTestIdWithText(window, 'settings-section');
+  await clickOnTestIdWithText(aliceWindow1, 'settings-section');
   // Click on privacy
-  await clickOnTestIdWithText(window, 'privacy-settings-menu-item');
+  await clickOnTestIdWithText(aliceWindow1, 'privacy-settings-menu-item');
   // Click set password
-  await clickOnTestIdWithText(window, 'set-password-button');
+  await clickOnTestIdWithText(aliceWindow1, 'set-password-button');
   // Enter password
-  await typeIntoInput(window, 'password-input', testPassword);
+  await typeIntoInput(aliceWindow1, 'password-input', testPassword);
   // Confirm password
-  await typeIntoInput(window, 'password-input-confirm', testPassword);
+  await typeIntoInput(aliceWindow1, 'password-input-confirm', testPassword);
   // Click Done
-  await clickOnMatchingText(window, 'Done');
+  await clickOnMatchingText(aliceWindow1, 'Done');
   // Check toast notification
   await waitForTestIdWithText(
-    window,
+    aliceWindow1,
     'session-toast',
     'Your password has been set. Please keep it safe.',
   );
   // Click on settings tab
-  await sleepFor(300);
-  await clickOnTestIdWithText(window, 'settings-section');
-  await clickOnTestIdWithText(window, 'recovery-phrase-settings-menu-item');
-  // Type password into input field
-  await typeIntoInput(window, 'password-input', testPassword);
+  await sleepFor(300, true);
+  await clickOnTestIdWithText(aliceWindow1, 'settings-section');
+  await clickOnTestIdWithText(
+    aliceWindow1,
+    'recovery-phrase-settings-menu-item',
+  );
+  await sleepFor(300, true);
+
+  // Type password into input field and validate it
+  await typeIntoInput(aliceWindow1, 'password-input', testPassword);
   // Click Done
-  await clickOnMatchingText(window, 'Done');
-  await clickOnTestIdWithText(window, 'settings-section');
+  await clickOnMatchingText(aliceWindow1, 'Done');
+
+  // check that the seed is visible now
+  await expectRecoveryPhraseToBeVisible(aliceWindow1, alice.recoveryPhrase);
+
+  // copy the seed phrase to the clipboard, also closes the dialog
+  await clickOnMatchingText(aliceWindow1, 'Copy');
+  // Note: I did not find a way to get the clipboard content from the page/window/nodeJS for now.
```

Comment:
That is wild. I could investigate this if you like?

Reply:
nope all good, just an idea for another time